webcompat/templates/issue-wizard-form.html

```diff
@@ -165,12 +165,10 @@ <h3 class="page-heading">Filing a web compatibility issue</h3>
           <div class="row mobile-col">
             <div class="form-element js-Form-group low with-validation-icons half">
               {{ form.browser.label(class_='form-label') }}
-              <small class="os-browser-error">Browser required</small>
```

Comment:
Do we want to remove the corresponding styles for `.os-browser-error`? 

https://github.com/webcompat/webcompat.com/search?l=CSS&q=os-browser-error

Reply:
Ah right, thanks! I've removed them